p2p/host/basic/basic_host_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 	"time"
 
+	autonat "github.com/libp2p/go-libp2p-autonat"
```

Comment:
I'd move this with this to the section 1 below (with go-libp2p.../identify and go-eventbus)